index.html

```diff
@@ -1099,5 +1098,110 @@ <h2>Internationalization Considerations</h2>
             guidance in the <a data-cite="vc-data-model/#internationalization-considerations">VC Data Model</a>.
          </p>
       </section>
+      <section class="normative">
+         <h2>IANA Considerations</h2>
+         <section id="media-types">
+            <h3>Media Types</h3>
+            <section id="schema-json-type">
+               <h2><code>application/schema+json</code></h2>
+               <p>
+                  This specification acknowledges the use of the <code>application/schema+json</code>
+                  Media Type specifically for identifying a [[JSON-SCHEMA]] with usage of the
+                  <a href="#jsonschema">JsonSchema</a> type, as registered in 
+                  <a href="#json-schema-specifications">JSON Schema specifications</a>.
+               </p>
+            </section>  
+            <section id="vc-jwt-schema-json-media-type">
+               <h2><code>application/vc+ld+jwt+schema+json</code></h2>
```

Comment:
I don't think you need this... in the case you are dereferencing a "JsonSchemaCredential" the response type will be:

1. `application/vc+ld+json` in the case of data integrity proof.
2. `application/vc+ld+json+jwt` or `application/vc+ld+json+sd-jwt` in the case of vc-jose-cose